lib/util.js

```diff
@@ -369,6 +369,30 @@ exports.getVideoDescription = (html) => {
     new Entities().decode(exports.stripHTML(description[1])) : '';
 };
 
+/**
+ * Get video media (extra information) from html
+ * 
+ * @param {String} body
+ * @return {Object}
+ */
+exports.getVideoMedia = (body) => {
+  let mediainfo = exports.between(body,
+    '<div id="watch-description-extras">',
+    '<div id="watch-discussion" class="branded-page-box yt-card">');
+  if (mediainfo === '') {
+    return {};
+  }
+  mediainfo = new Entities().decode(mediainfo);
+
+  const regexp = /<h4 class="title">([\s\S]*?)<\/h4>[\s\S]*?<ul.*?class=".*?watch-info-tag-list\">[\s\S]*?<li>([\s\S]*?)<\/li>/g;
+  const contentRegexp = /<a.*?>(.*?)<\/a>/;
+  const media = {};
+  while ((match = regexp.exec(mediainfo)) != null) {
```

Comment:
`match` is not defined. My suggestion from before was not correct sorry. Syntax is more like

```js
let match;
while ((match = ...) != null) {
```

Reply:
Ah yes, sorry. Never seen this syntax before, don't use while loops very often lol